data-serving/data-service/src/util/validate-env.ts

```diff
@@ -37,5 +38,11 @@ export default function validateEnv(): Readonly<{
             devDefault: true,
             default: false,
         }),
+        MAPBOX_GEOCODE_RATE_LIMIT_PER_SEC: num({
+            desc:
+                'number of requests per seconds allowed to mapbox geocode endpoint',
+            devDefault: 50,
```

Comment:
Will we hit this if we test ADI in dev?

Reply:
devDefault is only used when NODE_ENV is not production so that's for local dev and CI.
dev-curator.ghdsi.org will get 600, as well as prod.

Reply:
Of course -- MB, thanks.